nengo_viz/static/slidercontrol.js

```diff
@@ -0,0 +1,209 @@
+VIZ.SliderControl = function(min, max) {
```

Comment:
For now this would make sense. But in general I would favor dropping the `viz` prefix in filenames altogether. It is not really informative because everything is prefixed with `viz` and gives only clutter when listing the files. If at all needed I would prefer to use a directory instead of prefixing the files.

It might make sense to also provide some more structure with some directories given the growing number of files, for example move all components to their own subdirectory.